README.md

```diff
@@ -248,6 +248,24 @@ var logger = require('pino-http')({
 })
 ```
 
+Logging of requests' bodies is disabled by default since it can cause security risks such as having private user information (password, other GDPR-protected data, etc.) logged (and persisted in most setups). However if enabled, sensitive information can be redacted as per [redaction documentation](http://getpino.io/#/docs/redaction).
+
+Furthermore, logging more bytes does slow down throughput ([although the cost is negligible in most use cases](https://www.eschrade.com/page/the-cost-of-logging/)).
```

Comment:
Please remove this article link. I have strong evidence that says otherwise, at least for Node.js apps.

https://youtu.be/zja-_IYNrFc

cc @davidmarkclements

Reply:
va bene